src/main/java/io/netty/incubator/codec/quic/QuicheQuicServerCodec.java

```diff
@@ -98,6 +98,10 @@ protected QuicheQuicChannel quicPacketRead(ChannelHandlerContext ctx, InetSocket
                                                ByteBuf scid, ByteBuf dcid, ByteBuf token) throws Exception {
         ByteBuffer dcidByteBuffer = dcid.internalNioBuffer(dcid.readerIndex(), dcid.readableBytes());
         QuicheQuicChannel channel = getChannel(dcidByteBuffer);
+        if (channel == null && type == QuicPacketType.ZERO_RTT) {
+            // 0 rtt packet should obtain the server generated dcid with HmacSignQuicConnectionIdGenerator
```

Comment:
I would remove the mention of HmacSignQuicConnectionIdGenerator here as the user might use a different one. If it really depends on `HmacSignQuicConnectionIdGenerator ` we also need to do an instanceof check first. 
```suggestion
            // 0 rtt packet should obtain the server generated dcid
```

Reply:
I add a method 'boolean isIdempotent()' to replace instanceof check which I think is more common.